Nodejs/Product/Nodejs/TypingsAcquisition.cs

```diff
@@ -29,14 +29,30 @@
 
 namespace Microsoft.NodejsTools {
     internal class TypingsAcquisition {
-        private const string TypingsTool = "typings";
-        private const string TypingsToolVersion = "1.0.5"; // Lock to stable version of the 'typings' tool with known behavior.
+        private const string TypingsTool = "ntvs_install_typings";
         private const string TypingsToolExe = TypingsTool + ".cmd";
+
         private const string TypingsDirectoryName = "typings";
 
         private static SemaphoreSlim typingsToolGlobalWorkSemaphore = new SemaphoreSlim(1);
 
         /// <summary>
+        /// Full path to the local copy of the typings acquision tool.
+        /// </summary>
+        private static string LocalTypingsToolPath {
+            get {
+                string codeBase = System.Reflection.Assembly.GetExecutingAssembly().CodeBase;
+                var uri = new UriBuilder(codeBase);
+                var path = Uri.UnescapeDataString(uri.Path);
+                var root = Path.GetDirectoryName(path);
+
+                return Path.Combine(
+                    root,
+                    "TypingsAquisitionTool");
+            }
```

Comment:
Any packages installed to this directory likely won't get cleaned up during uninstall, so as discussed, let's move to just publishing the package to npm